azavea/commonhealth-data-map
# Updates repository contents with most recent EAC Dashboard changes

## Overview

Selectively copies the most up-to-date components of the EAC Dashboard repository into this one. Includes application, data processing, tile generation, etc.

### Checklist


```console
$ chan added "A feature with no bugs. [#21](https://github.com/azavea/commonhealth-data-map/pull/21)"
```

- [x] Unnecessary `console.log()`s are removed

### Notes

- There were a bunch of React build warnings in the logs that aren't failing the build because `CI=1` wasn't being set; I added a comment to https://github.com/azavea/commonhealth-data-map/issues/22#issuecomment-622452094 to make sure we account for fixing that later.
- It appears as though the website build times have gone up a bit. Earlier in the day they were exhausting the Netlify 30 minute build limit, but now I'm starting to wonder if that was a noisy neighbor effect. Either way, I went partway down a path to try and minimize the build times, which led to the removal of the `mb-util` phase.

## Testing Instructions

- Follow the instructions in the **Processing data** section of the `README`; ensure that the data update procedures complete successfully
- Execute the `./scripts/build-website` script locally; ensure that it completes successfully and generates the website in `build/`
- Review the GitHub Actions [build log](https://github.com/azavea/commonhealth-data-map/runs/636820922) for automated data updates
- Review the [deploy preview](https://deploy-preview-21--commonhealth-data-map.netlify.app/cases); ensure that the site operates as expected and that the vector tiles load properly

Closes #9
